.gitignore

```diff
@@ -57,6 +57,7 @@ env/
 env[23]/
 .dotrun.json
 .venv
+venv2/
```

Comment:
why is this necessary?

Reply:
It keeps the postgres things separate from dotrun's virtual environment.

Reply:
maybe the naming for the venv can be a bit more descriptive in this case, something like `pg-venv`?

Reply:
yeah that's a better name.